kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -16,6 +16,8 @@
         <span class="middle-breadcrumb landscape" v-for="crumb in topicCrumbs">
           <a v-link="topicLink(crumb.id)">{{ crumb.title }}</a>
         </span>
+        <!-- Had to add a space or else the breadcrumbs would jump.-->
+        <span class="middle-breadcrumb landscape"> {{' ' + title}}</span>
```

Comment:
@66eli77 It's to show the full hierarchy. I'ts pretty standard to include the current page even though the page might already have the current page title elsewhere.